packages/react/src/__tests__/template.test.tsx

```diff
@@ -378,7 +375,7 @@ it('should pass return type from onTemplateEvent back to component', async () =>
         key: 'button',
         component: 'Profile',
         props: {
-          onClick: event(action('go-back')),
+          onClick: action('go-back'),
```

Comment:
nice much cleaner. This will also support JSX right? So I can do `onClick={action('go-back')}` ?

Reply:
yes! it is natively supported, so you can pass this value as a prop with/without JSX and it will work just fine (it is not wrapped in event anymore behind the scenes).